### ISSUE TITLE
QEventDispatcherUNIXPrivate(): Unable to create thread pipe: Too many open files

### ISSUE THREAD

**Issue by loretoparisi** [ariya/phantomjs#14180]
``` shell
QEventDispatcherUNIXPrivate(): Unable to create thread pipe: Too many open files
2016-04-06T17:57:58 [FATAL] QEventDispatcherUNIXPrivate(): Can not continue without a thread pipe
PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.
Abort trap: 6
```

I get this error when aborting (Ctrl-C) or calling `phantom.exit()` during a `page.render` or `page.open` on multiple page. It happens sometimes on the `https://github.com/ariya/phantomjs/blob/master/examples/render_multi_url.js` example.

**Comment by loretoparisi** [comment]
Also it gets

```
Apr  6 18:07:24 macbookproloreto.local phantomjs[87218] <Error>: CGContextSetFont: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
Apr  6 18:07:24 macbookproloreto.local phantomjs[87218] <Error>: CGContextSetTextPosition: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
Apr  6 18:07:24 macbookproloreto.local phantomjs[87218] <Error>: CGContextShowGlyphsWithAdvances: invalid context 0x0. This is a serious error. This application, or a library it uses, is using an invalid context  and is thereby contributing to an overall degradation of system stability and reliability. This notice is a courtesy: please fix this problem. It will become a fatal error in an upcoming update.
```

**Comment by Vitallium** [comment]
This issue will be fixed with the update to Qt 5.7.

More information is available on this page: https://wiki.qt.io/New_Features_in_Qt_5.7
>https://wiki.qt.io/New_Features_in_Qt_5.7

**Comment by loretoparisi** [comment]
@Vitallium thank you!
